chart/templates/deployment-budget.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ .Release.Name }}-budget
+spec:
+  minAvailable: 1
+  selector:
+    matchLabels:
+      app: {{ .Release.Name}}
+      tier: web
```

Comment:
These labels are not correct for all resources. The Deployment resource has

```
app: svc-questionnaires
```

Reply:
Ok I'll hardcode it in there.

Reply:
Fixed.

Reply:
Edit; nvm: i changed the hardcoded one to use .Release.Name